guide/keyv/README.md

```diff
@@ -63,10 +63,10 @@ Keyv exposes a familiar [Map](https://developer.mozilla.org/en-US/docs/Web/JavaS
 
 ## Application
 
-Although Keyv can assist in any scenario where you need key-value data, we will focus on setting up a per-guild prefix configuration using Sqlite.
+Although Keyv can assist in any scenario where you involve key-value data, we will focus on setting up a per-guild prefix configuration using Sqlite.
```

Comment:
need is more specific than involve, and semantically correct here, no need to change